cypress/support/commands.ts

```diff
@@ -109,3 +109,12 @@ Cypress.Commands.add('requestBase64Image', (url: string) => {
       return Cypress.Blob.blobToBase64String(blob);
     });
 });
+
+Cypress.Commands.add('keyboardControls', (triggerKeys: any = {}, count = 1) => {
+  const body = cy.get('body');
+
+  for (let i = 0; i < count; i++) {
+    body.trigger('keydown', triggerKeys)
```

Comment:
Why do we need both down and up?

Reply:
The linter is suggesting `Do not assign the return value of a Cypress command`.
I don't think the body is going to change for this tests, but there's some cases where it may happens.